src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
-'use strict';
+const employeeListItems = document.querySelectorAll('ul li');
 
-// write code here
+function getSalary(salaryString) {
+  return parseInt(salaryString.replace(/[$,]/g, ''), 10);
+}
+
+function sortList(listItems) {
+  const sortedList = Array.from(listItems).sort((a, b) => {
+    return (
+      getSalary(b.getAttribute('data-salary')) -
+      getSalary(a.getAttribute('data-salary'))
```

Comment:
The sorting logic is currently sorting in descending order by salary. If the requirement is to sort in ascending order, you should switch the order of `a` and `b` in the `getSalary` function calls.